ksql-functional-tests/src/main/java/io/confluent/ksql/test/model/PostConditionsNode.java

```diff
@@ -41,6 +48,28 @@ public PostConditions build() {
         .toArray(Matcher[]::new);
 
     final Matcher<Iterable<DataSource<?>>> sourcesMatcher = hasItems(matchers);
-    return new PostConditions(sourcesMatcher);
+    final Pattern blackListPattern = topics.build();
+
+    return new PostConditions(sourcesMatcher, blackListPattern);
+  }
+
+  @JsonIgnoreProperties(ignoreUnknown = true)
+  public static final class TopicsNode {
+
+    private final String blackList;
+
+    public TopicsNode(
+        @JsonProperty("blacklist") final String blackList
+    ) {
+      this.blackList = blackList == null ? "" : blackList;
+    }
+
+    Pattern build() {
```

Comment:
```suggestion
    Pattern buildBlacklist() {
```

Reply:
`build` is a common name used across all node types.